docs/_pages/fsharp.md

```diff
@@ -0,0 +1,45 @@
+---
+title: F# Usage
+permalink: /fsharp/
+layout: single
+classes: wide
+sidebar:
+  nav: "sidebar"
+---
+
+### F# usage
+
+We encourage F# developers to try out [Faqt](https://github.com/cmeeren/Faqt/), an assertion library inspired by FluentAssertions and made specifically for F#.
+
+FluentAssertions is made with C# in mind. While it can be used from F#, the experience will not be as intended. In particular:
```

Comment:
```suggestion
FluentAssertions was made with C# in mind. While it can be used from F#, the experience will not always be as intended.

For a more native F# experience we encourage F# developers to try out [Faqt](https://github.com/cmeeren/Faqt/), an assertion library inspired by FluentAssertions and made specifically for F#.

Some of the awkwardness you might hit with Fluent Assertions when using F# is:
```

Reply:
Might I suggest replacing "more native" with "better"?

Looks good except for the choice of the word "native". It's not technically incorrect, but fluent syntax isn't more or less native than the documented workarounds needed to make FluentAssertions work with F#.

Reply:
Sure, especially all the adjectives used could probably be tuned.

Reply:
I think it's good except for "native" 👍 Is it easy for you to adjust your commit suggestion so I can commit it here, or would you rather I commit it myself locally and push?

Reply:
adjusted

Reply:
In the last sentence, shouldn't it be `FluentAssertions` instead of `Fluent Assertions`?

Reply:
Fixed